lib/msal-core/src/utils/Constants.ts

```diff
@@ -91,19 +89,22 @@ export enum TemporaryCacheKeys {
     LOGIN_REQUEST = "login.request",
     RENEW_STATUS = "token.renew.status",
     URL_HASH = "urlHash",
-    ANGULAR_LOGIN_REQUEST = "angular.login.request",
-    INTERACTION_STATUS = "interaction.status"
+    ANGULAR_LOGIN_REQUEST = "angular.login.request"
 }
 
 export enum PersistentCacheKeys {
     IDTOKEN = "idtoken",
-    CLIENT_INFO = "client.info",
-    ADAL_ID_TOKEN = "adal.idtoken",
+    CLIENT_INFO = "client.info"
+}
+
+export enum ErrorCacheKeys {
+    LOGIN_ERROR = "login.error",
     ERROR = "error",
-    ERROR_DESC = "error.description",
-    LOGIN_ERROR = "login.error"
+    ERROR_DESC = "error.description"
 }
 
+export const INTERACTION_STATUS = "interaction.status";
```

Comment:
Not sure why this was moved out

Reply:
I separated the existing `PersistentCacheKeys` into `PersistentCacheKeys` and `ErrorCacheKeys` (this is to separate error handing from UAA and the enable cache wipe for error messages as appropriate). Since `INTERACTION_STATUS` did not belong in both, separated it.

Reply:
I think it should be moved back into TemporaryCacheKeys, this is just to organize the Constants file. We don't use the enums as arrays anyway, so it should be fine. I would like to prevent using one-off constants, and rather group constants into the relevant enums or objects.

Reply:
Done